syntax="proto3";

message CreateRoomC2S {
    int32 roomSize = 1;
}